rocketmq-console/src/test/java/org/apache/rocketmq/console/service/impl/MessageTraceServiceImplTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.rocketmq.console.service.impl;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import lombok.SneakyThrows;
+import org.apache.rocketmq.client.QueryResult;
+import org.apache.rocketmq.common.message.MessageExt;
+import org.apache.rocketmq.console.config.RMQConfigure;
+import org.apache.rocketmq.console.model.MessageTraceView;
+import org.apache.rocketmq.console.model.trace.MessageTraceGraph;
+import org.apache.rocketmq.console.util.JsonUtil;
+import org.apache.rocketmq.tools.admin.MQAdminExt;
+import org.assertj.core.util.Lists;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
run this test fails in my computer, MockitoJUnitRunner init fail, it.s fine if remove @RunWith(MockitoJUnitRunner.class)
![image](https://user-images.githubusercontent.com/14816818/124523784-17d92980-de2b-11eb-98bb-b27a01bb4bd3.png)

Reply:
@RunWith(MockitoJUnitRunner.class) already removed.